core/ui/EditTemplate.tid

```diff
@@ -1,16 +1,26 @@
 title: $:/core/ui/EditTemplate
 
+\define actions()
```

Comment:
possibly, this should be renamed to **edit-add-actions** or some such and could be reused in [$:/core/ui/Buttons/save](https://github.com/Jermolene/TiddlyWiki5/blob/master/core/ui/EditToolbar/save.tid#L6-L12) to replace the actions there